bitnami/jasperreports/README.md

```diff
@@ -86,14 +86,15 @@ The command removes all the Kubernetes components associated with the chart and
 | `image.pullPolicy`      | JasperReports image pull policy                                                                               | `IfNotPresent`          |
 | `image.pullSecrets`     | Specify docker-registry secret names as an array                                                              | `[]`                    |
 | `jasperreportsUsername` | JasperReports user                                                                                            | `jasperadmin`           |
+| `existingSecret`        | Name of existing secret containing JasperReports credentials                                                  | `""`                    |
```

Comment:
If this only affects SMTP, could you change the name of the variable?
```suggestion
| `smtpExistingSecret`        | Name of existing secret containing JasperReports credentials                                                  | `""`                    |
```

Reply:
Hi @mr-onion-2,

the readme generator takes the variables of the values.yaml, if you updated that file, the readme will be update too.

Reply:
Hi @Mauraza 

I've moved the _'Ignored if existingSecret is provided'_ note onto the @param line so that it is included in the readme, for consistency

Reply:
Hi @mr-onion-2 , 
If you are in the account, this type of annotation does not appear in the rest of the charts, because of using a descriptive variable.

Reply:
Hi @Mauraza

Sorry I'm not sure what you mean. Do you need me to change something else?

I note the 'VIB verify' action in the CI pipeline failed due to an unrelated timeout 

Thanks

Reply:
Hi @Mauraza

The secret contains the keys `jasperreports-password` (required) and `smtp-password` (optional). I think calling it `smtpExistingSecret` is confusing.

I have split `existingSecret` into two - `jasperreportsExistingSecret` and `smtpExistingSecret` for clarity, though they most likely will be the same.